### PR TITLE
[GUILD] Typescript `MarketingTypography`

### PR DESCRIPTION
**Pull request by BeirlaenAaron**
This PR converts the `MarketingTypography` component to TS

### CODE REVIEW

File: src/@types/types.ts [hunk 1]
```diff
@@ -0,0 +1,3 @@
+import { FunctionComponent } from 'react';
+
+export type GenericComponent<P = {}> = FunctionComponent<P>;
```
Comment (qubis741) [inline src/@types/types.ts:3]: this is not supposed to be in your PR. rebase it pls

**Review by KristofColpaert — APPROVED** [review]
👌

**Review by eniskraasniqi1 — APPROVED** [review]
MC'd ✅

**Review by stefaandevylder — APPROVED** [review]

**Review by JorenSaeyTL — APPROVED** [review]

**Comment by BeirlaenAaron** [comment]
Sooo this was merged but the PR status didn't update 🤔

**Pull request closed without merging**